scannerlib/repeatconfig/repeat_config.go

```diff
@@ -35,6 +35,7 @@ const (
 	gidWildcard      = "$gid"
 	homeDirWildcard  = "$home"
 	portWildcard     = "$port"
+	shellWildcard    = "$shell"
```

Comment:
Add a comment to scan_instructions.proto about this wildcard and what this replaces the text with.